azure-pipelines.yml

```diff
@@ -23,22 +26,22 @@ stages:
     variables:
       publisher: $(SonarSourceITPublisher)
     steps:
+    
     - task: Npm@1
       displayName: 'Update npm'
+      env:
+        NPM_CONFIG_//repox.jfrog.io/artifactory/api/npm/:_authToken: $(ARTIFACTORY_PRIVATE_READER_ACCESS_TOKEN)
```

Comment:
Using `npm config set ..` at the top level didn't work, so the token is passed to each task that needs it